src/input/mod.rs

```diff
@@ -2991,13 +3000,17 @@ mod tests {
         let mut suppressed_keys = HashSet::new();
 
         let screenshot_ui = ScreenshotUi::new(Default::default());
+        let access_dialog_ui = AccessDialogUi::new();
         let disable_power_key_handling = false;
 
         // The key_code we pick is arbitrary, the only thing
         // that matters is that they are different between cases.
 
         let close_key_code = Keycode::from(close_keysym.raw() + 8u32);
-        let close_key_event = |suppr: &mut HashSet<Keycode>, mods: ModifiersState, pressed| {
+        let close_key_event = |suppr: &mut HashSet<Keycode>,
+                               mods: ModifiersState,
+                               pressed,
+                               access_dialog_ui: &AccessDialogUi| {
```

Comment:
Hm, why can it capture screnshot_ui but not this?

Reply:
Just a leftover, had it as mutable reference at some point.

Reply:
removed the unnecessary change